src/test/java/io/vertx/ext/awssdk/AsyncHttpClientTest.java

```diff
@@ -33,32 +40,32 @@
   private static final String HOST = "localhost";
   private static final String SCHEME = "http";
 
-  @Rule
-  public Timeout rule = Timeout.seconds(1);
-
-  @Before
+  @BeforeEach
   public void setUp() {
     vertx = Vertx.vertx();
     server = vertx.createHttpServer();
     client = new VertxNioAsyncHttpClient(vertx);
   }
 
-  @After
-  public void tearDown(TestContext ctx) {
+  @AfterEach
+  public void tearDown(VertxTestContext ctx) {
       if (server == null) {
           return;
       }
-      server.close(ctx.asyncAssertSuccess());
+      server.close(res -> {
+          assertTrue(res.succeeded());
+          ctx.completeNow();
+      });
   }
 
   @Test
-  public void testGet(TestContext ctx) {
-    Async async = ctx.async();
+  @Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
```

Comment:
such low timeout might be an issue on travis CI that runs slower than on your laptop